Samples/Samples.iOS/Info.plist

```diff
@@ -55,6 +55,10 @@
 	<string>1.0</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Access to your location is required for cool things to happen!</string>
+    <key>NSCalendarsUsageDescription</key>
+    <string>Add calendar events</string>
+    <key>NSRemindersUsageDescription</key>
+    <string>Add calendar events</string>
```

Comment:
Let's update these strings to be more useful

Reply:
Changes amended in newest commit